app/middleware/fedache.rb

```diff
@@ -0,0 +1,37 @@
+require 'fixed_length/parser'
+require 'fixed_length/column'
+require 'faraday_middleware/response_middleware'
+
+class Fedache < FaradayMiddleware::ResponseMiddleware
+  extend FixedLength::Column
+
+  column :routing_number, length: 9
```

Comment:
`Fedache` is a pun right? Hehe, nice.

Yea agreed. Middleware serves to manipulate an HTTP request and pass the manipulated result to the next slice of middleware in the stack. The logic here is more relevant to parsing a bank data string - you could find a home for it in `/lib` pretty easily I think.